src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2929,6 +2929,25 @@ public void testIndentationLineBreakVariableDeclaration()
         verifyWarns(checkConfig, fileName, expected);
     }
 
+    @Test
+    public void testLambdaInEnum() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+        checkConfig.addProperty("tabWidth", "4");
+        checkConfig.addProperty("basicOffset", "4");
+        checkConfig.addProperty("braceAdjustment", "0");
+        checkConfig.addProperty("caseIndent", "4");
+        checkConfig.addProperty("throwsIndent", "4");
+        checkConfig.addProperty("forceStrictCondition", "false");
+        checkConfig.addProperty("lineWrappingIndentation", "4");
```

Comment:
Please make this the same order that config comment in input file is.

Reply:
done

Reply:
Resolved.